src/main/webapp/assets/modules/contest-detail/contestDetailDirective.js

```diff
@@ -114,6 +114,42 @@ techlooper
       }
     };
   })
+    .directive('tabManagerContestDetail', function () {
+      return {
+        restrict: "E",
+        replace: true,
+        templateUrl: "modules/contest-detail/tabManager.html",
+        link: function (scope, element, attr, ctrl) {
+        }
+      };
+    })
+    .directive('funnelManager', function () {
+      return {
+        restrict: "E",
+        replace: true,
+        templateUrl: "modules/contest-detail/funnelManager.html",
+        link: function (scope, element, attr, ctrl) {
+        }
+      };
+    })
+    .directive('contestContentDetails', function () {
+      return {
+        restrict: "E",
+        replace: true,
+        templateUrl: "modules/contest-detail/contestContentDetails.html",
+        link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.<br>'scope' is defined but never used.